app/frontend/src/submissions/views/SubmissionsHome.vue

```diff
@@ -681,6 +681,33 @@ export default {
         this.validateWellFields(errors);
       }
     },
+    /**
+     * @desc Users must fill in one of three "Well Location" fields (well location address, legal description, parcel identifier),
+     * To count as correct, one of these three sections has to be filled in.
+     * @summary validates Well Location section of form 
+     * @returns {Boolean} a valid section has been filled out
```

Comment:
I believe currently the isWellLocationFilled function will return undefined. We could update the return type here to be undefined or we could explicitly return true or false and leave the return type as Boolean.

Reply:
@fergmac 
Thanks for the catch, my first attempt involved returning a boolean, but since the errors object both prevents advancing AND shows user feedback, the functionality was later changed.

Line 688 is now removed :)